src/libstd/process.rs

```diff
@@ -1167,6 +1167,24 @@ pub fn abort() -> ! {
     unsafe { ::sys::abort_internal() };
 }
 
+/// Returns the OS-assigned process identifier associated with this process.
+///
+/// # Examples
+///
+/// Basic usage:
+///
+/// ```no_run
+/// use std::process:current_pid;
+///
+/// println!("My pid is {}", current_pid());
+/// ```
+///
+///
+#[unstable(feature = "getpid", issue = "44971", reason = "recently added")]
+pub fn current_pid() -> u32 {
```

Comment:
I think `i64` or `u64` would be more prudent.

Reply:
We already use u32 elsewhere: https://doc.rust-lang.org/std/process/struct.Child.html#method.id

Reply:
Yep, that's why I chose u32. Although I think usize or u32 would probably make more sense for both.